CHANGELOG.rst

```diff
@@ -12,18 +12,28 @@ Internal Changes
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **Knowledge Graph**: Improve access tokens management
+* **Knowledge Graph**: Add migration importing Composite Plans into the Triples Store
+* **Knowledge Graph**: Prevent from data corruption caused by duplicate Project creation dates
 
 **Bug fixes**
 
 * **Sessions**: Handle expiring Gitlab tokens
 * **Gateway**: Do not remove Redis clients on logout
 * **Gateway**: Address security vulnerabilities
+* **Knowledge Graph**: Fix vulnerabilities in Graph Services' docker images
+* **Knowledge Graph**: Handle cases when Plan Invalidation Time is wrong
+* **Knowledge Graph**: Made Cross-Entity Search API to sort in a case-insensitive way
+* **Knowledge Graph**: Treat query phrase given to the Cross-Entity Search API as a String
+* **Knowledge Graph**: Return all inactive Projects from the User's-Projects API
```

Comment:
Add functionality to return all inactive Projects from User-Project API ?

Reply:
It was actually a bug on the API that made it return only the first 20 inactive projects. Now it returns all of them. Maybe it's ok if read in the context of the `Bugfix` section.

Reply:
I'm not sure what you're proposing here; maybe just make it:

`Bgufix: Return all inactive Projects from the User's-Projects API`

Reply:
Or:
```suggestion
* **Knowledge Graph**: Return all inactive Projects from the User's-Projects API (previously only returned 20)
```